src/Nethermind/Nethermind.Network.Stats/Model/InitiateDisconnectReason.cs

```diff
@@ -8,9 +8,9 @@ namespace Nethermind.Stats.Model;
 /// </summary>
 public enum InitiateDisconnectReason : byte
 {
-    IncomingConnectionRejectedTooManyPeer,
+    TooManyPeer,
```

Comment:
`TooManyPeers` I think it should be plural?

Reply:
Done